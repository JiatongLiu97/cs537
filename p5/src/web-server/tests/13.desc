Only dynamic requests (checking stat_process)
