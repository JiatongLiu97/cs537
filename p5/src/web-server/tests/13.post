killall -q -u $USER -s SIGKILL server ; rm -rf /dev/shm/$TEST_SHM_NAME
