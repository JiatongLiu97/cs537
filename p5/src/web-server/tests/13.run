bash tests/stat_test.sh 8       12      20      1     "/output.cgi?0.1"
#                       threads buffers clients loops requests
