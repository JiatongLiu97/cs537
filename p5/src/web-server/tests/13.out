Total requests: 20; Total static: 0; Total dynamic: 20
